net.pcsx2.PCSX2.metainfo.xml

```diff
@@ -20,6 +20,7 @@
       <li>Increase or decrease the game speed by using the built-in frame-limiter</li>
       <li>Ability to record in full HD with the built-in video recorder (F12 using the GSdx plugin)</li>
     </ul>
+    <p>Copy your PS2 BIOS files to the default BIOS search path `$HOME/.var/app/net.pcsx2.PCSX2/config/PCSX2/bios`.</p>
```

Comment:
```suggestion
    <p>Copy your PS2 BIOS files to the default BIOS search path <code>$HOME/.var/app/net.pcsx2.PCSX2/config/PCSX2/bios</code>.</p>
```

Reply:
This is not rendering correctly in flathub.org nor Discover, I've opened #59.

Reply:
I reported this to Flathub here: https://github.com/flathub/website/issues/361

Feel free to report it to Discover.

Reply:
For the time being it's probably fair to just use quotes.

Reply:
Absolutely, more just a long term nice to get working thing.

It may actually be a bug in the underlying appstream utils flathub uses.